bscript/interpreter/README.md

```diff
@@ -0,0 +1,42 @@
+interpreter
+========
+
+[![ISC License](http://img.shields.io/badge/license-ISC-blue.svg)](http://copyfree.org)
+[![GoDoc](https://pkg.go.dev/badge/github.com/libsv/go-bt/bscript/interpreter?utm_source=godoc)](http://godoc.org/github.com/libsv/got-bt/bscript/interpreter)
+
+Package interpreter implements the an interpreter for the bitcoin transaction language.  There is
+a comprehensive test suite.
+
+This package has intentionally been designed so it can be used as a standalone
+package for any projects needing to use or validate bitcoin transaction scripts.
+
+## Bitcoin Scripts
+
+Bitcoinsv provides a stack-based, FORTH-like language for the scripts in
```

Comment:
replace bitcoinsv with just bitcoin - bsv is just the ticker, bitcoin is bsv (btc, bch and other bitcoin forks don't haven't even restored full script functionality anyway...)